### PR TITLE
off logs

### PR DESCRIPTION
**Pull request by andreymheilasystems**

### CODE REVIEW

File: bundler/logger.go [hunk 1]
```diff
@@ -9,9 +10,16 @@ import (
 	"github.com/orchestd/log/bzerolog"
 	"github.com/orchestd/sharedlib/consts"
 	"go.uber.org/fx"
+	"os"
 )
 
 func LoggerFxOption() fx.Option {
+	turnoffLogger := os.Getenv("turnoffLogger")
```
Comment (orchestd) [inline bundler/logger.go]: disabledLogger

File: bundler/logger.go [hunk 2]
```diff
@@ -9,9 +10,16 @@ import (
...
+	if turnoffLogger == "true" {
+		return fx.Options(
+			fx.Provide(emptyLogger.NewEmptyLogger),
```
Comment (orchestd) [inline bundler/logger.go:20]: it looks like in this case you turn off line 58 as well
		trace.TraceInfoContextExtractorFxOption(),
is this ok ??

File: bundler/trace.go [hunk 3]
```diff
@@ -8,11 +8,18 @@ import (
 	"github.com/orchestd/sharedlib/consts"
 	"github.com/orchestd/trace/bjaeger"
 	"go.uber.org/fx"
+	"os"
 )
 
 type Tracer opentracing.Tracer
 
 func TracerFxOption() fx.Option {
+	turnoffTrace := os.Getenv("turnoffTrace")
```
Comment (orchestd) [inline bundler/trace.go]: disableTracer

**Review by orchestd — CHANGES_REQUESTED** [review]

File: bundler/logger.go (hunk 2 above)
Comment (andreymheilasystems) [reply bundler/logger.go:20]: yes, off everything about tracer

**Review by orchestd — APPROVED** [review]

**Pull request merged**